src/index.js

```diff
@@ -158,7 +158,7 @@ const initPlayer = (params, embed) => {
 const loadPlayer = (params, resolve, reject) => {
   checkParams(params);
 
-  const refNode = params.refNode;
+  const refNode = resolveRefNode(params.refNode);
```

Comment:
What if the `document.querySelector` does not return a dom element?

Reply:
Whoops, good point.

Reply:
Updated. The `resolveRefNode` will now throw if it does not find a node instead of just in one case.